index.js

```diff
@@ -16,10 +17,10 @@ ${title}
 ${favicon}
 ${css}
 ${head}
+${script}
 </head>
 <body>
 ${body}
-${script}
```

Comment:
Agreed! Just wanted to confirm that was as desired before complicating things with the extra logic. I'll make that change.